standard/expressions.md

```diff
@@ -4161,7 +4161,7 @@ equality_expression
     ;
 ```
 
-> *Note*: Lookup for the right operand of the `is` operator must first test as a *type*, then as an *expression* which may span multiple tokens. In the case where the operand is an *expreesion*, the pattern expression must have precedence at least as high as *shift_expression*. *end note*
+> *Note*: Lookup for the right operand of the `is` operator must first test as a *type*, then as an *expression* which may span multiple tokens. In the case where the operand is an *expression*, the pattern expression must have precedence at least as high as *shift_expression*. *end note*
```

Comment:
We should consider reverting this change, and doing a global search and replace of expression => expreesion as a far more fun term.

Reply:
I did a global search on GitHub. This is the only instance.